src/Compilers/VisualBasic/Test/Symbol/SymbolDisplay/ObjectDisplayTests.vb

```diff
@@ -179,6 +179,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
         End Sub
 
         <Fact>
+        Public Sub TextForEscapedStringLiterals()
+            Dim literal = SyntaxFactory.Literal(ChrW(&H2028) & "x") ' U+2028 is a line separator
+            Assert.Equal("ChrW(8232) & ""x""", literal.Text)
+            literal = SyntaxFactory.Literal(ChrW(&HDBFF)) ' U+DBFF is a unicode surrogate
+            Assert.Equal("ChrW(56319)", literal.Text)
+        End Sub
```

Comment:
Consider testing `ObjectDisplay.FormatLiteral` directly. Perhaps:
```
Assert.Equal("...",
    ObjectDisplay.FormatLiteral(
        ChrW(&HD83C) & ChrW(&HDFC8),
        ObjectDisplayOptions.UseQuotes Or ObjectDisplayOptions.EscapeNonPrintableCharacters))
```
And a similar test for C# `ObjectDisplayTests`:
```
Assert.Equal("...",
    ObjectDisplay.FormatLiteral("\ud83c", ObjectDisplayOptions.EscapeNonPrintableCharacters));
Assert.Equal("...",
    ObjectDisplay.FormatLiteral("\ud83c\udfc8", ObjectDisplayOptions.EscapeNonPrintableCharacters));
```